Mathlib/FieldTheory/AbsoluteGaloisGroup.lean

```diff
@@ -52,20 +53,20 @@ instance absoluteGaloisGroup.commutator_closure_isNormal :
 
 /-- The topological abelianization of `absoluteGaloisGroup`, that is, the quotient of
   `absoluteGaloisGroup` by the topological closure of its commutator subgroup. -/
-def absoluteGaloisGroup_abelianization :=
-  G_K K ⧸ Subgroup.topologicalClosure (commutator (G_K K))
+def absoluteGaloisGroup_abelianization := TopologicalAbelianization (G_K K)
```

Comment:
Similarly this could also potentially be made an abbrev.

Reply:
Also, I believe that the name should be `absoluteGaloisGroupAbelianization` instead.